include/pmacc/particles/ParticlesBase.kernel

```diff
@@ -229,7 +229,14 @@ struct KernelFillGapsLastFrame
                 uint32_t const
             )
             {
-                pb.getSuperCell( superCellIdx ).setSizeLastFrame( counterParticles );
+                // there is no need to add a zero to the global memory
+                if( counterParticles != 0 )
```

Comment:
I'm not sure `counterParticles` is a good name. First, as the word 'counter' is also an adjective, for me it seems unnatural to put it first, since here 'particles' is a dependant word. Also, is there any reason a name like 'numParticlesLastFrame' (or any variation with 'number' instead of 'counter') cannot be used here?

The first consideration also applies for `counterGaps`.